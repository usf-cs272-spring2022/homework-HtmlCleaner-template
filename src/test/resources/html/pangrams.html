<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">

<html>

<head>
    <title>Pangrams</title>
    <meta http-equiv="Content-Type" content="text/html;charset=utf-8">
    <sTyLE type="text/css">
        body
        {
            font-family: sans-serif;
        }
        
        .elephant
        {
            color: green;
        }
    </style>
</head>

<BoDY>

<h1>The quick, brown fox jumps over a lazy dog.</H1>

<a Href="http://commons.wikimedia.org/wiki/File:Sunny_Fox.jpg"><img src="http://upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Sunny_Fox.jpg/320px-Sunny_Fox.jpg" alt="this is not an elephant"></a>
    
<p
      >DJs flock by when MTV ax quiz prog. Junk MTV quiz graced by fox &#8212; whelps. Bawds jog, flick <b    >quartz</b>, vex nymphs. Waltz, bad nymph, for quick jigs vex! &copy; Fox nymphs grab quick-jived waltz. Painful zombies quickly watch a jinxed graveyard.</P>

<script      src="http://widgets.twimg.com/j/2/widget.js" type="text/javascript">console.log("Anything unrelated to elephants is irrelphant.");</SCRIPT>
<scRipt type="text/javascript">
new TWTR.Widget({
  version: 2,
  type: 'profile',
  rpp: 4,
  interval: 30000,
  width: 250,
  height: 300,
  theme: {
    shell: {
      background: '#333333',
      color: '#ffffff'
    },
    tweets: {
      background: '#000000',
      color: '#ffffff',
      links: '#4aed05'
    }
  },
  features: {
    scrollbar: false,
    loop: false,
    live: false,
    behavior: 'all'
  }
}).render().setUser('usfca').start();
</script     >

<h2>Brick quiz whangs &laquo; jumpy veldt fox.</h2>
    
<p class="elephant">Bright vixens jump; dozy fowl <i>quack</i>. Quick wafting zephyrs vex bold Jim. Quick zephyrs blow, vexing daft Jim. <A name="elephant"     
     href="http://www.blindtextgenerator.com/lorem-ipsum">How quickly daft jumping zebras vex.</a> Cozy lummox gives smart squid who asks for job pen.</p>

<hr width="50%">

<ul>
    <li>Quick, Baz, get my woven flax <u>jodhpurs</u>! 
    <li>"Now fax quiz Jack!" my brave ghost pled. 
    <li>Five quacking zephyrs jolt my wax bed. 
</ul>

<table width="100%" cellspacing="0" cellpadding="2" border="1">
<tr>
    <td>Cozy sphinx waves quart jug of &reg; bad milk.</TD>
    <td>A very bad quack might jinx zippy fowls.</td>
</tr>
<tr>
    <td colspan="2">Few quips galvanized the mock jury box.</td>
</tr>
</table>

<p>&#182; <a HREF="http://en.wikipedia.org/wiki/List_of_pangrams" name="elephant">Quick brown dogs jump over the lazy fox</a>. The jay, pig, fox, zebra, and my wolves quack! <span style="color: red;">How razorback-jumping &frac34; frogs can level six piqued gymnasts!</span></p>

</body>

</html>
