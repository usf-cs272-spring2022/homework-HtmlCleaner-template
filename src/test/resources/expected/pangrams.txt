







The quick, brown fox jumps over a lazy dog.


    
DJs flock by when MTV ax quiz prog. Junk MTV quiz graced by fox — whelps. Bawds jog, flick quartz, vex nymphs. Waltz, bad nymph, for quick jigs vex! © Fox nymphs grab quick-jived waltz. Painful zombies quickly watch a jinxed graveyard.




Brick quiz whangs « jumpy veldt fox.
    
Bright vixens jump; dozy fowl quack. Quick wafting zephyrs vex bold Jim. Quick zephyrs blow, vexing daft Jim. How quickly daft jumping zebras vex. Cozy lummox gives smart squid who asks for job pen.




    Quick, Baz, get my woven flax jodhpurs! 
    "Now fax quiz Jack!" my brave ghost pled. 
    Five quacking zephyrs jolt my wax bed. 




    Cozy sphinx waves quart jug of ® bad milk.
    A very bad quack might jinx zippy fowls.


    Few quips galvanized the mock jury box.



¶ Quick brown dogs jump over the lazy fox. The jay, pig, fox, zebra, and my wolves quack! How razorback-jumping ¾ frogs can level six piqued gymnasts!



